dali/dfuplus/dfuplus.cpp

```diff
@@ -599,20 +599,25 @@ int CDfuPlusHelper::spray()
     const char* srcfile = globals->queryProp("srcfile");
 
     bool nowait = globals->getPropBool("nowait", false);
+    bool nolocal = globals->getPropBool("nolocal", false);
 
     MemoryBuffer xmlbuf;
 
     if(srcxml == nullptr)
     {
         if(srcfile == nullptr)
             throw MakeStringException(-1, "srcfile not specified");
-        if(srcip == nullptr) {
+        if(srcip == nullptr)
+        {
+            srcip = "127.0.0.1";
```

Comment:
I think removing this will behave differently.  Previously I think it would have been remapped by checkLocalDaFileSvr() to the external ip of your local machine.  That is needed for the spray to work.
You could use GetCachedHostName() instead.
(I haven't checked the behaviour).